src/main/java/com/google/cloud/genomics/utils/grpc/ReadStreamIterator.java

```diff
@@ -35,8 +35,6 @@
  *
  * Includes complex retry logic to upon failure resume the stream at the last known good start
  * position without returning duplicate data.
```

Comment:
This sentence makes me want to read it multiple times.  How about the following:

*This function integrates complex retry logic, which upon encountering a failure will resume the stream at the last known valid start position, without having any duplicated data in returned stream.*